CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Unreleased
 
+- [breaking] Support Storybook 10
```

Comment:
Will be a new major release to support Storybook 10.

It's what the Storybook folks recommend for plugins (although this project isn't technically a plugin). Also makes sense to me, so it's really clear what version of Storybook is known to work.